www/config.yml

```diff
@@ -1197,8 +1197,9 @@ chapters:
   pro_network: Python Software Foundation Meetup Pro Network
   twitter: pyladiesvan
   website: vancouver
-- email: vienna@pyladies.com
-  image: pyladies_dc_atl_sea_tor.png
+- email: pyladies.vienna@gmail.com
```

Comment:
We'd like chapters to use the official PyLadies.com email address instead of personal gmail. Does anyone in your group have access to it?

Reply:
Thank you for the review. We do not have access to the pyladies.com one.
We have been actively using the private gmail one for several years already.

If referencing a @pyladies.com one is required, we can envision having an inbound mail forwarding set up, but we are not going to change the email for outbound messages out of a sudden while quite a lot of PyLadies from our chapter know this one.

Reply:
Our policy is that you conduct your activities using the PyLadies.com email address. This is described in https://kit.pyladies.com/en/latest/prospective/index.html#for-prospective-organizers

The form linked in the page can be used to request access to the existing PyLadies chapter, so I would suggest that you follow the process of reactivating the Vienna chapter.

Reply:
We have taken over the chapter from previous organizers which also didn't use official email. Currently since we are communicating with our PyLadies local chapter members via the presented gmail account, we will not be changing it. If this is a problem for updating the details on the website, then I will close this PR. Unfortunately such a big change is simply not currently possible for our chapter and activities.